packages/terra-date-picker/src/react-datepicker/year_dropdown_options.jsx

```diff
@@ -113,12 +116,12 @@ export default class YearDropdownOptions extends React.Component {
 
   render () {
     let dropdownClass = classNames({
```

Comment:
This could be an array since these still need to have the module bindings applied.

Reply:
updated.